.circleci/config.yml

```diff
@@ -141,16 +141,11 @@ jobs:
       - run: md test-results
 
       - run:
-          name: "List rules"
-          command: bin\run.cmd scanner:rule:list
+          name: "Unpackaged sanity test"
```

Comment:
Can you add comments about the purpose of this run, same applies to `packaged`

Reply:
Will do.